src/app/components/task/EditTaskDialog.js

```diff
@@ -238,13 +238,13 @@ class EditTaskDialog extends React.Component {
         ))}
         <div style={{ marginTop: units(1) }}>
           <Button onClick={this.handleAddValue.bind(this)}>
-            {this.props.intl.formatMessage(messages.addValue)}
+            <FormattedMessage id="props" defaultMessage="props" {...messages} />
           </Button>
           <Button
             onClick={this.handleAddOther.bind(this)}
             disabled={this.state.hasOther}
           >
-            {this.props.intl.formatMessage(messages.addOther)}
+            <FormattedMessage id="props" defaultMessage="props" {...messages} />
```

Comment:
Same here, `id` and `defaultMessage` should not be the `"props"` string. They should have the actual values that are in `messages.addOther`.